### PR TITLE
Add VS Code go plugin

### PR DESCRIPTION
**Pull request by tolusha**
Signed-off-by: Anatoliy Bazko <abazko@redhat.com>

### Reference issue
https://github.com/eclipse/che/issues/12145

### What does this PR do?
This PR introduces VS Code GO plugin [1]

[1] https://marketplace.visualstudio.com/items?itemName=ms-vscode.Go

### CODE REVIEW

**Comment by tolusha** [comment]
https://github.com/eclipse/che/issues/12145

File: plugins/ms-vscode.go/0.9.2/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,64 @@
+FROM eclipse/che-theia-endpoint-runtime:nightly
```
Comment (benoitf) [inline plugins/ms-vscode.go/0.9.2/Dockerfile]: hello, FYI missing copyright header

Comment (benoitf) [reply plugins/ms-vscode.go/0.9.2/Dockerfile]: question: Dockerfile should go in che-theia, no ?

Comment (tolusha) [reply plugins/ms-vscode.go/0.9.2/Dockerfile]: you are right

**Comment by tolusha** [comment]
It is there
https://hub.docker.com/r/eclipse/che-remote-plugin-go-1.10.7/tags

**Comment by benoitf** [comment]
OK thx

**Comment by benoitf** [comment]
@tolusha : how was it published ? i don't see it in che-theia repository

**Comment by tolusha** [comment]
@benoitf I asked @riuvshin to publish it to be able to merge PR

**Comment by benoitf** [comment]
Well it bypass any code review :-/

**Comment by tolusha** [comment]
https://github.com/eclipse/che-theia/pull/84

**Pull request merged**